README.md

```diff
@@ -0,0 +1,69 @@
+![TNW](https://user-images.githubusercontent.com/37639594/105220349-42c31000-5b60-11eb-9061-95509ac37a01.png)
+
+Clone of the The Next Web website page putting emphasis on how it behaves differently depending on the size of the screen.
+
+## Built With
+
+- HTML5
+- CSS
+- Media querry
+
+
+## Live Demo
+
+HEAD TO
+[TNW PROJECT](https://ashraffares.github.io/TNW/)
+=======
+
+
+
+
+
+### Prerequisites
+Knowledge about HTML and CSS.
+HTML Semantic elements.
+CSS flex property.
+CSS grid property.
+Media querry
+
+
+
+
+## Authors
+
+👤 **Fares Ashraf**
+
+- twitter:https://twitter.com/Fares09301164
+- GitHub: https://github.com/ashraffares/http-ashraffares.github.io-
+- linkedin:https://www.linkedin.com/in/fares-ashraf-382a35176/
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+- Clone this repository with git clone```https://github.com/ashraffares/TNW.git`` using your terminal or command line.
```

Comment:
- [ ] [REQUIRED] You should check your closing backticks for this line. They should be the same number as the opening ones. With this done, the correct formatting should be applied.